### PR TITLE
{Monitor} log analysis bump up SDK

### PR DESCRIPTION
**Pull request by kairu-ms**
Closed https://github.com/Azure/azure-cli/issues/17532
Closed https://github.com/Azure/azure-cli/issues/17231
Closed https://github.com/Azure/azure-cli/issues/17230
Closed https://github.com/Azure/azure-cli/issues/16207

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Monitor

File: src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py [hunk 1]
```diff
@@ -273,9 +273,9 @@ def test_monitor_log_analytics_workspace_recover(self, resource_group):
         self.kwargs.update({
             'table_name': 'Syslog'
         })
-        # Disable checks due to service issue: https://github.com/Azure/azure-rest-api-specs/issues/12407
+
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py:276]: Is this issue fixed now?

File: src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py [hunk 2]
```diff
@@ -422,22 +422,21 @@ def test_monitor_log_analytics_workspace_data_export(self, resource_group, accou
                      '--destination {sa_id_1} --enable -t "SecurityEvent Heartbeat"',
                      checks=[
                      ])
-        # Disable this test because msrest cannot deserialize content-type "text/plan".
-        # with self.assertRaisesRegexp(HttpResponseError, 'You have exceeded the number of allowed export rules in your workspace'):
-        #     self.cmd('monitor log-analytics workspace data-export create -g {rg} --workspace-name {workspace_name} -n {data_export_name_2} '
-        #              '--destination {sa_id_1} --enable -t {table_name}',
-        #              checks=[
-        #              ])
+        with self.assertRaisesRegexp(HttpResponseError, 'you can create 10 export rules to 10 different destinations'):
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py:425]: No deserialize issue now?

**Review by jsntcy — APPROVED** [review]

File: src/azure-cli/requirements.py3.Darwin.txt [hunk 3]
```diff
@@ -54,7 +54,7 @@ azure-mgmt-iothub==2.0.0
 azure-mgmt-iothubprovisioningservices==0.2.0
 azure-mgmt-keyvault==9.0.0
 azure-mgmt-kusto==0.3.0
-azure-mgmt-loganalytics==8.0.0
+azure-mgmt-loganalytics==10.0.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.Darwin.txt:57]:
```suggestion
azure-mgmt-loganalytics==11.0.0
```

File: src/azure-cli/requirements.py3.Linux.txt [hunk 4]
```diff
@@ -54,7 +54,7 @@ azure-mgmt-iothub==2.0.0
 azure-mgmt-iothubprovisioningservices==0.2.0
 azure-mgmt-keyvault==9.0.0
 azure-mgmt-kusto==0.3.0
-azure-mgmt-loganalytics==8.0.0
+azure-mgmt-loganalytics==10.0.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.Linux.txt:57]:
```suggestion
azure-mgmt-loganalytics==11.0.0
```

File: src/azure-cli/requirements.py3.windows.txt [hunk 5]
```diff
@@ -54,7 +54,7 @@ azure-mgmt-iothub==2.0.0
 azure-mgmt-iothubprovisioningservices==0.2.0
 azure-mgmt-keyvault==9.0.0
 azure-mgmt-kusto==0.3.0
-azure-mgmt-loganalytics==8.0.0
+azure-mgmt-loganalytics==10.0.0
```
Comment (kairu-ms) [inline src/azure-cli/requirements.py3.windows.txt:57]:
```suggestion
azure-mgmt-loganalytics==11.0.0
```

File: src/azure-cli/setup.py [hunk 6]
```diff
@@ -96,7 +96,7 @@
     'azure-mgmt-iothubprovisioningservices~=0.2.0',
     'azure-mgmt-keyvault==9.0.0',
     'azure-mgmt-kusto~=0.3.0',
-    'azure-mgmt-loganalytics~=8.0.0',
+    'azure-mgmt-loganalytics~=10.0.0',
```
Comment (kairu-ms) [inline src/azure-cli/setup.py:99]:
```suggestion
    'azure-mgmt-loganalytics~=11.0.0',
```

File: src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py (hunk 2 above)
Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/monitor/tests/latest/test_monitor_log_analytics_workspace.py:425]: Yes, it's resolved.

File: src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_azure_monitor.yaml [hunk 7]
```diff
@@ -2232,13 +2315,13 @@ interactions:
...
-      - AZURECLI/2.25.0 azsdk-python-mgmt-loganalytics/8.0.0 Python/3.7.2 (Windows-10-10.0.19041-SP0)
+      - AZURECLI/2.26.0 azsdk-python-mgmt-loganalytics/7.0.0b1 Python/3.8.10 (Windows-10-10.0.19043-SP0)
     method: POST
     uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/hdicli-000001/providers/Microsoft.OperationalInsights/workspaces/testws000003/sharedKeys?api-version=2020-08-01
   response:
     body:
-      string: "{\r\n  \"primarySharedKey\": \"xxxxxx\",\r\n
-        \ \"secondarySharedKey\": \"xxxxxx\"\r\n}"
+      string: "{\r\n  \"primarySharedKey\": \"/Qzedotpq3RQnmlZYGdpDjsns1gwXjCDzqAI2PI5uVGAjTn9ziiPhawfbu84zm3fiQDUWDYpWxjraMJ0fmAe/Q==\",\r\n
+        \ \"secondarySharedKey\": \"/97lASBaRs9YSnxqSH2/VzhGezedPSDJO6aqoosHlz29JeRy/q103fErskAd61Hj9j6XjstG2CYluesqlpyhDQ==\"\r\n}"
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_azure_monitor.yaml:2324]:
```suggestion
      string: "{\r\n  \"primarySharedKey\": \"xxxxxx\",\r\n
        \ \"secondarySharedKey\": \"xxxxxx\"\r\n}"
```

File: src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_azure_monitor.yaml [hunk 8]
```diff
@@ -2276,7 +2359,7 @@ interactions:
       code: 200
       message: OK
 - request:
-    body: '{"workspaceId": "a807639a-11a7-49e6-8e29-8aa9b21bc9ed", "primaryKey": "xxxxxx"}'
+    body: '{"workspaceId": "c18a5cc1-8df0-44ba-a8c5-d595510c30d8", "primaryKey": "/Qzedotpq3RQnmlZYGdpDjsns1gwXjCDzqAI2PI5uVGAjTn9ziiPhawfbu84zm3fiQDUWDYpWxjraMJ0fmAe/Q=="}'
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/recordings/test_hdinsight_azure_monitor.yaml:2362]:
```suggestion
    body: '{"workspaceId": "c18a5cc1-8df0-44ba-a8c5-d595510c30d8", "primaryKey": "xxxxxx"}'
```

**Review by zhoxing-ms — APPROVED** [review]

**Review by Juliehzl — APPROVED** [review]
LGTM for code owner part

File: src/azure-cli/azure/cli/command_modules/sql/_help.py [hunk 9]
```diff
@@ -1211,6 +1211,11 @@
             --event-hub-target-state Disabled
 """
 
+helps['sql server ms-support'] = """
+type: group
+short-summary: Manage a server's Microsoft support operations.
+"""
+
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/sql/_help.py:1218]: Already covered in #18520

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**